src/pages/DirectMessage/components/DMChattingList/DMInput/DMInput.tsx

```diff
@@ -0,0 +1,62 @@
+import * as S from "./DMInput.Styles"
+import MessageProfile from "../../MessageGroupList/MessageProfile"
+import { useEffect, useState } from "react"
+import useSendMessage from "../../../hooks/useSendMessage"
+import SendIcon from "@mui/icons-material/Send"
+import { AUTH_API } from "@/apis/Api"
+
+interface DMInputProps {
+  othersId: string
+}
+
+const DMInput = ({ othersId }: DMInputProps) => {
+  const [sendingMessage, setSendingMessage] = useState("")
+  const { mutate: sendMessage } = useSendMessage()
+  const [myProfileImg, setMyProfileImg] = useState("")
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target
+    setSendingMessage(value)
+  }
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    const messageSubmission = {
+      message: sendingMessage,
+      receiver: othersId,
+    }
+    sendMessage(messageSubmission)
+    setSendingMessage("")
+  }
+
+  const getMyProfileImg = async () => {
+    const { image } = await AUTH_API.get(`/auth-user`)
+      .then((res) => res.data)
+      .catch((e) => {
+        console.log(e, "이미지 요청 실패")
+      })
+    setMyProfileImg(image)
+  }
+
+  useEffect(() => {
+    getMyProfileImg()
+  }, [])
+
+  return (
+    <S.DMInputLayout>
+      <S.DMInputForm onSubmit={handleSubmit}>
+        <MessageProfile profileImg={myProfileImg} />
+        <S.DMInputItem
+          placeholder="메시지를 입력해주세요"
+          onChange={handleInputChange}
+          value={sendingMessage}
+        />
+        <S.DMSendButton>
+          <SendIcon />
+        </S.DMSendButton>
+      </S.DMInputForm>
+    </S.DMInputLayout>
+  )
+}
+
+export default DMInput
```

Comment:
[ P1 ] 현재 인증된 즉 로그인한 사용자의 이미지를 불러오려는 목적이라면 useAuthUserStore에서 꺼내오는게 좋지 않을까요!

```suggestion
   const { user: authUser } = useAuthUserStore()
  const { image } = authUser

  return (
    <S.DMInputLayout>
      <S.DMInputForm onSubmit={handleSubmit}>
        <MessageProfile profileImg={image} />
        <S.DMInputItem
          placeholder="메시지를 입력해주세요"
          onChange={handleInputChange}
          value={sendingMessage}
        />
        <S.DMSendButton>
          <SendIcon />
        </S.DMSendButton>
      </S.DMInputForm>
    </S.DMInputLayout>
  )
}

export default DMInput
```

Reply:
> [ P1 ] 현재 인증된 즉 로그인한 사용자의 이미지를 불러오려는 목적이라면 useAuthUserStore에서 꺼내오는게 좋지 않을까요!

넵 저도 저렇게 하는게 훨씬 좋긴한데 새로고침 시 세션스토리지에는 user의 id만 남아서 일단 저렇게 구현했습니다ㅜ

Reply:
저도 뭔가 새로고침시 user 의 id만 상태에 남아있는게 다음에 걸려서 생각해봤는데

- navbar에 세션스토리지가 있을시 무조건 패치를 실행해 user 스토리지에 값을 담는 로직을 추가하는건 어떨까요??

어차피 현재 nav가 존재하는곳은 전부 login user가 사용가능한 위치일것 같아요

Reply:
요거에 대한 자세한건 내일 이야기 나눠보시죠!